sources/imperative/reader/ReaderAPI.swift

```diff
@@ -71,6 +71,46 @@ extension CSVReader {
             try self.init(configuration: configuration, buffer: buffer, decoder: decoder)
         }
     }
+
+    public enum Stream {
+        case stdin
+    }
```

Comment:
What file should this enum go in?

Reply:
If we decide to go in this direction, I would add it into `ReaderCollections.swift` and maybe rename the file to `ReaderIO.swift` or something like that.